docker/ctAutocomplete/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:17-alpine
```

Comment:
Hmm... node LTS is at 16... why node 17?

Reply:
🔧